inlang/source-code/create-project/src/tryAutoGenProjectSettings.ts

```diff
@@ -66,16 +65,6 @@ export async function tryAutoGenProjectSettings(args: {
 		}
 	}
 
-	if (errors.length > 0) {
-		try {
-			await args.nodeishFs.rm(projectFilePath)
-		} catch {
-			/* ignore failing file removal */
-		}
-
-		return { warnings, errors }
-	}
-
 	return { warnings, errors, settings }
 }
 
```

Comment:
@felixhaeberle why is `autoGenProject` exported and not included in `tryAutoGenProjectSettings`?

Reply:
for testing purposes – but we can also test it through `tryAutoGenProjectSettings`